src/api/NotificationsAssistant.ts

```diff
@@ -98,22 +102,33 @@ export default class NotificationsAssistant{
 	async ClearNotifications(){
 
 		const pending = await Plugins.LocalNotifications.getPending()
-		Plugins.LocalNotifications.cancel(pending) //clear all the pending notifications
+		if(pending.notifications.length != 0){
```

Comment:
Check for typescript compiler warnings. != is considered bad style and will cause a warning. Change to !==